fairytale.md

```diff
@@ -16,3 +16,7 @@ The average Squonk is 7 meters tall and screams loud.
 >  What happened between it and the *Drowsloks*?
 
 `Well... your turn now.`
+
```

Comment:
Hi @ju5t1nas ! Could you please move your story above the line
```
>  Add some lines to the story. Can be anything. Inspiration?  
```
that is, around line 10?

More info will come later.